src/acl.c

```diff
@@ -2188,9 +2187,24 @@ void aclCommand(client *c) {
             struct redisCommand *cmd = dictGetVal(de);
             if (cmd->flags & CMD_MODULE) continue;
             if (cmd->acl_categories & cflag) {
-                addReplyBulkCString(c,cmd->name);
+                addReplyBulkCBuffer(c, cmd->fullname, sdslen(cmd->fullname));
```

Comment:
why not addReplyBulkSdsNoFree?

Reply:
Not sure it's worth making a copy of the same code for nofree sds, which has been used extensively elsewhere in the code to avoid sds being freed.